docs/reference/changelog-r2020.md

```diff
@@ -6,6 +6,8 @@ Released on XXX.
   - New Features
     - Added a [Mesh](mesh.md) node allowing to use external 3D file in Webots ([#1419](https://github.com/cyberbotics/webots/pull/1419)).
     - Added two new functions to get internal PROTO node fields: [`wb_supervisor_node_get_from_proto_def`](supervisor.md#wb_supervisor_node_get_from_proto_def) and [`wb_supervisor_node_get_proto_field`](supervisor.md#wb_supervisor_node_get_proto_field) ([#1331](https://github.com/cyberbotics/webots/pull/1331)).
+  - Enhancements
+    - **Improved [Track](track.md) animationTexture field to make it more transparent to the user.**
```

Comment:
Note that the field name is wrong.

```suggestion
    - **Improved [Track](track.md) `textureAnimation` field to make it more transparent to the user.**
```

Then we decided to add a link to the GitHub PR. Could you please add it as it is done in the previous entry?

Personally in the log I would write what will change for the user, for example something like:
``Improved Track `textureAnimation` field so that it automatically takes the world basic time stamp into consideration.``